lib/chef-dk/helpers.rb

```diff
@@ -47,6 +47,22 @@ def stderr
       $stderr
     end
 
+    #
+    # Locates the habitat directories
+    #
+
+    def habitat_install?
+      File.exist?(habitat_chefdk_location)
+    end
+
+    def habitat_bin_dir
```

Comment:
What do you think about a more generic-named `bin_dir` (or `dk_bin_dir` or something) that encapsulates the branching logic of determining where the executables are? I sense that we could perform the `if habitat_install?` "what kind of install am I?" check once and then everywhere that needs to know can rely on the helper.

Reply:
Yup, we can call it that, the main thing here is to have functions to detect those installation methods. Which from you comment I think you agree is the right way to go. 

I'll update it.